mass_mailing_custom_unsubscribe/static/src/js/unsubscribe.js

```diff
@@ -0,0 +1,286 @@
+/* Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+/* This JS module replaces core AJAX submission because it is impossible
+ * to extend it as it is currently designed. */
```

Comment:
We explained in https://github.com/OCA/social/pull/402#issuecomment-511707513 that this file is almost a copy+paste from upstream, and that's why it's so ugly.

So, to avoid making readers and linters crazy, let's add this here:

```suggestion
 * to extend it as it is currently designed. It is almost a copy+paste from
 * upstream, to allow easier version/patch updates, so linter is disabled. */
 /* eslint-disable */
```